solana-programs/scripts/deploy.sh

```diff
@@ -0,0 +1,271 @@
+#!/bin/bash
+
+set -e
+
+function airdrop_till {
```

Comment:
i think we could make the params/fn sig here a little clearer for this useful fn, either externally like `airdrop_to_1_until_balance_reaches_2` or internally by having local vars equal to $1 and $2